asserts/pool_test.go

```diff
@@ -107,6 +110,28 @@ func (s *poolSuite) SetUpTest(c *C) {
 	c.Assert(err, IsNil)
 	s.rev2_2222 = a.(*asserts.TestOnlyRev)
 
+	// sequence-forming
+
+	a, err = s.hub.Sign(asserts.TestOnlySeqType, map[string]interface{}{
+		"n":        "1111",
+		"sequence": "1",
+		"id":       "one",
+		"dev-id":   "developer1",
+		"revision": "5",
+	}, nil, "")
+	c.Assert(err, IsNil)
+	s.seq1_1111 = a.(*asserts.TestOnlySeq)
+
+	a, err = s.hub.Sign(asserts.TestOnlySeqType, map[string]interface{}{
+		"n":        "1111",
+		"sequence": "2",
+		"id":       "one",
+		"dev-id":   "developer1",
+		"revision": "7",
```

Comment:
having this higher than 5 is hiding some bugs I think,   isAssertionNewer need to consider pinned vs not pinned, and compare sequence as well if unpinned:  for updates we need tests for various scenarios

* pinned with  same revision or  newer revision
* unpinned  with    same sequence, same revision,  or same sequence higher revision,   or  higher latest sequence (incoming revision should not matter)